pkg/controller/migration/convert/core.go

```diff
@@ -9,6 +15,10 @@ func handleCore(c *components, install *Installation) error {
 		return ErrIncompatibleCluster{"only DATASTORE_TYPE=kubernetes is supported at this time"}
 	}
 
+	if err = handleFelixNodeMetrics(c, install); err != nil {
```

Comment:
I don't think we want handlers calling other handlers. Can you move this up to where `handleCore` is called from?

Reply:
Glad you added it to the caller, but looks like you forgot to remove it here